CONTRIBUTING.md

```diff
@@ -1,38 +1,115 @@
-## How to contribute
+## Table of Contents
+- [Introduction](#introduction)
+- [1) How to make and test code changes](#1-how-to-make-and-test-code-changes)
+- [2) Developer Process for Contributing](#2-developer-process-for-contributing)
+- [3) Release Process](#3-release-process)
```

Comment:
Is there a way to change these bullet points to a numbered list or maybe as an indented numbered list? For example:

- Introduction
  1. How to make and test code changes
  2. Developer Process for Contributing
  3. Release Process